ci-operator/step-registry/api-job/k8sgpt/api-job-k8sgpt-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+openai_token=$(cat "/var/run/vault/tests-private-account/openai-token")
+
+k8sgpt version
+k8sgpt auth add -m gpt-3.5-turbo -b openai -p ${openai_token} | tee ${ARTIFACT_DIR}/result || true
+k8sgpt analyze -aed | tee -a ${ARTIFACT_DIR}/result || true
+
+log=`cat ${ARTIFACT_DIR}/result`
+
+if [[ log =~ "No problems detected" ]]; then
+    cat >"${ARTIFACT_DIR}/result.xml" <<EOF
```

Comment:
@jianzhangbjz suggest to change `${ARTIFACT_DIR}/result.xml` to `${ARTIFACT_DIR}/k8sgpt-result.xml`